app/controllers/events_controller.rb

```diff
@@ -0,0 +1,35 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = current_user.events.build(event_params)
+    return unless @event.save
+
+    @event.content.capitalize!
+    redirect_to current_user
+  end
+
+  def show
+    @attendance = current_user.attendances.build
+    @event = Event.find(params[:id])
+    @invited = @event.attendees.where('pending')
+    @accepted = @event.attendees.where('accepted')
+    @rejected = @event.attendees.where('rejected')
+    @users = User.where.not(id: current_user.id).to_a.reject do |user|
+      @invited.include?(user) || @accepted.include?(user) || @rejected.include?(user)
+    end
```

Comment:
*OPTIONAL*
I suggest you define a method in the `User` model and call that method here `current_user.`

Reply:
hey, the app does that, just go to http://localhost:3000/events, o make click in "Events", not "My Events", to see those events separated in upcoming and past